src/backend/distributed/deparser/deparse_type_stmts.c

```diff
@@ -378,7 +378,11 @@ AppendColumnDefList(StringInfo str, List *columnDefs)
 static void
 AppendColumnDef(StringInfo str, ColumnDef *columnDef)
 {
-	Oid typeOid = LookupTypeNameOid(NULL, columnDef->typeName, false);
+	int32 typmod;
+	Oid typeOid;
```

Comment:
Variables not initialised, need to be zero initialised according to code convention.
```suggestion
	int32 typmod = 0;
	Oid typeOid = InvalidOid;
```